content/docs/music.md

```diff
@@ -5,17 +5,40 @@ next: "/docs/build"
 nextTitle: "Building Your Game"
 ---
 
-GB Studio is internally using [GBT Player](https://github.com/AntonioND/gbt-player). Additional documentation and sample MOD files can be found in its own repository.
+GB Studio is internally using [GBT Player](https://github.com/AntonioND/gbt-player) *(GameBoy Tracker)*. Additional documentation and sample MOD files can be found in its own repository.
 
 GBT Player reads and plays back .MOD files. You can use software such as [OpenMPT](https://openmpt.org/) or [MilkyTracker](https://milkytracker.titandemo.org/) to create .MOD tracker music. Of course, you can use other software that can edit .mod files (like [BassoonTracker](https://www.stef.be/bassoontracker/), [ProTracker](https://16-bits.org/pt.php), etc.)
 
+# How do I start?
+
+GameBoy music is made using a tracker, a *computer music* oriented program.
+
+1. Download a tracker (recommended [OpenMPT](https://openmpt.org/))
+2. In a blank Gbstudio project, find the file `assets/music/template.mod` and open it with your tracker.\
+ **You must edit this file for the right GameBoy instruments.**
+3. Start deleting notes from the song (some trackers support drag selection).
+4. Select your instument and press letter keys on the keyboard to make notes.
+5. Read your trackers documentation, and find tutorials on using trackers in general.
+6. Keep testing your song.mod in the GBstudio emulator, as it will always sound different.
+
 When done, add your MOD files to the `assets/music` folder of your project.
 
-# Important Limitations
+## How to use a tracker
+Once you add notes using the letter keys, you should see\
+`D-5 01 ---` in your tracker.\
+That's the **note**, **instrument**, and **effect**.\
+`C` is the volume effect,\
+Change `---` to `C40` for loud, and `C00` to mute, or any [hexadecimal](https://www.mathsisfun.com/hexadecimals.html) inbetween.\
+Most trackers use the arrow keys for navigation.\
+There are 64 notes per pattern, but you can keep adding more patterns.
```

Comment:
## Getting Started with a Tracker
Enter your tracker's edit mode and press keys on your keyboard. Once you have added a note, you should see\
`D-5`, `01`, and `---` on one of the 64 lines.\
These are the **note**, **instrument**, and **effect** parameters.\
You can change your note's volume by entering `C` in the effect area.\
Change `C--` to `C40` for maximum volume or `C00` to mute your note.\
There are 64 notes (or "ticks") per pattern. You can't have more than 64 ticks in GBT Player.
Add a pattern to your song to keep your song going.

Reply:
I tried to clean this up a bit, but more importantly to show `C--` instead of `---` when you enter C, and to remove the part about hexadecimals. Explanation below